lib/core-integration/src/Test/Integration/Scenario/API/Shelley/StakePools.hs

```diff
@@ -735,6 +737,7 @@ spec = describe "SHELLEY_STAKE_POOLS" $ do
                 saturation `shouldSatisfy` (any (> 0))
 
         it "contains pool metadata" $ \ctx -> do
+            updateMetadataSource ctx "direct"
```

Comment:
Hmm, what's the default of the tests?

Reply:
It used to check on-chain by default before, but now its `none`... that's what it is also now in the integration test cluster.
Yes, I was surprised it worked, with this check passing:
```
metadataActual
          `shouldSatisfy` (`Set.isSubsetOf` metadataPossible)
```
so it turns out that:

```
> (fromList []) `isSubsetOf` (fromList [1])
True
```

Reply:
Lolo